keyboards/yanghu/unicorne/f411/info.json

```diff
@@ -1,5 +1,3 @@
 {
```

Comment:
![image](https://github.com/qmk/qmk_firmware/assets/16520359/1875b50e-6583-4a62-9324-bb7d88aba6b4)

These need checking as i can see this doesnt use a development board, so this change is incorrect.

Reply:
> Given this, I can only assume others are incorrect too.
re-evaluation done as per commit 7755f0e